app/services/setting_registry.rb

```diff
@@ -113,7 +113,6 @@ def load
       # Creating missing records as we operate over the DB model while updating the setting
       _new_db_record(definition).save(validate: false)
       definition.updated_at = nil
-      definition.value ||= definition.default
```

Comment:
I don't quite understand this bit. Why do you create DB model definitions but don't set a value? Wouldn't it be easier to not save them at all then?

Reply:
It would, but I didn't want to go that far :shrug: